daemon/logger/jsonfilelog/jsonfilelog_test.go

```diff
@@ -54,6 +56,49 @@ func TestJSONFileLogger(t *testing.T) {
 	}
 }
 
+func TestJSONFileLoggerWithTags(t *testing.T) {
+	cid := "a7317399f3f857173c6179d44823594f8294678dea9999662e5c625b5a1c7657"
+	cname := "test-container"
+	tmp, err := ioutil.TempDir("", "docker-logger-")
+	if err != nil {
+		require.NoError(t, err)
+	}
```

Comment:
You don't need to check for `err != nil` first, that's what `require.NoError()` does. Just remove the if condition.

Reply:
Done.